### PR TITLE
RIA-8035 - Removed listing status check for CMR and added logs for de…

### PR DESCRIPTION
**Pull request by robin-ieong**
Removed listing status check for CMR and added logs for debugging

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by MiguelMartinRojas — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java [hunk 1]
```diff
@@ -139,7 +139,6 @@ public DynamicList buildHearingChannelDynmicList(List<HearingChannel> hearingCha
 
     public boolean isCaseManagementReview(ServiceData serviceData) {
         return isHmcStatus(serviceData, HmcStatus.LISTED)
-            && isHearingListingStatus(serviceData, ListingStatus.FIXED)
```
Comment (MiguelMartinRojas) [inline src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java:142]: Why did you delete this?

**Review by rajeshthuraiyur — DISMISSED** [review]
LGTM

**Review by MiguelMartinRojas — APPROVED** [review]
